qemu/tests/cfg/virtio_scsi_cdrom.cfg

```diff
@@ -68,12 +73,39 @@
                     image_name_stg22 = images/storage22
                     image_name_stg23 = images/storage23
                     image_name_stg24 = images/storage24
-                    image_name_stg25 = images/storage25
                     image_size_equal = 1G
```

Comment:
Hi qinwang, if the param image_size_equal = 1G is unused?

Reply:
(1/1) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.virtio_scsi_cdrom.with_installation.cdrom.extra_cdrom_ks.aio_threads.multi_disk_install.q35: PASS (1051.41 s)

 (1/2) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.virtio_scsi_cdrom.with_installation.cdrom.extra_cdrom_ks.aio_threads.multi_disk_install: PASS (690.56 s)
 (2/2) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.virtio_scsi_cdrom.with_installation.cdrom.extra_cdrom_ks.aio_threads.multi_disk_install.q35: PASS (898.38 s)